src/components/sidebar/components/sidebar-nav/sidebar-nav-menu-item/style.module.css

```diff
@@ -48,7 +48,10 @@
     z-index: 1;
   }
 
-  /* Hover styles take precedence over focus & active styles and should be defined them */
+  /*
+  Hover styles take precedence over focus & active styles, so hover styles
+  should be after defined them.
```

Comment:
```suggestion
  should be defined after them.
```

Reply:
LOL oops I was trying to fix the typo and I think I made it worse via another typo 🤣